custom_components/rivian/__init__.py

```diff
@@ -182,6 +185,12 @@ async def _update_api_data(self):
             vijson = await vehicle_info.json()
             self._login_attempts = 0
 
+            if self._wallboxes or self._wallboxes is None:
+                resp = await self._api.get_registered_wallboxes()
+                if resp.status == 200:
+                    wallboxes = (await resp.json())["data"]["getRegisteredWallboxes"]
```

Comment:
developer debug line here would be helpful
this would be similar to what is done with the vehicle polling response:

```suggestion
-                    wallboxes = (await resp.json())["data"]["getRegisteredWallboxes"]
+                    wbjson = await resp.json()
+                    _LOGGER.debug(wbjson)
+                    wallboxes = wbjson["data"]["getRegisteredWallboxes"]
```